webapp/web/configure/tags.css

```
@@ -73,3 +73,15 @@
   border: 1px dashed var(--default-border-color);
   background-color: var(--lighter-background-color);
 }
+
+.text-edit.unsaved::before,
+.tag.unsaved::before {
+  font-family: "Font Awesome 5 Free";
+  content: "\f853";
+  display: inline-block;
+  padding-right: 3px;
+  vertical-align: middle;
+  color: var(--orange);
+  font-weight: 900;
+  margin-right: 3px;
+}
```

nit: github complained about needing a new line at the end of this file
```suggestion
}

```